### ISSUE TITLE
Fully support the Hyper X Alloy Origins 

### ISSUE THREAD

**Issue by ZillionPrey** [antonpup/Aurora#2247]
Please add support for the following device: HyperX Alloy Origins

**Device Name:** Alloy Origins

**Device Website:** https://www.hyperxgaming.com/

This keyboard is available and working with OpenRGB if I select the Alloy Elite RGB Keyboard. Tho three keys are not lighting up on a german keyboard layout: < # and the FN key. Everything else is working perfectly.

**Comment by xQwexx** [comment]
Hi if you update Aurora with the latest dev build https://ci.appveyor.com/project/antonpup/aurora/builds/39567830/artifacts
you could use Aurora with OpenRGB, where this keyboard supported

**Comment by ZillionPrey** [comment]
Yeah I'm already using a dev build it only has the Alloy Elite RGB in it. This profile does work but it doesn't light up the keys described above.

**Comment by tavallon** [comment]
I've been using OpenRGB and  'Alloy Elite RGB' device on Aurora for my Alloy Origins for a long time, and Fn key is the only one that doesn't light up. I do know that OpenRGB just had a [couple fixes](https://gitlab.com/CalcProgrammer1/OpenRGB/-/commit/173862666e36ab892905d44554257c1f0273bcb8) for Alloy Origins on [Nov 26](https://gitlab.com/CalcProgrammer1/OpenRGB/-/commit/44d7e27404db6239319310d6c00ad9d3091b389c), so maybe it's possible to use the new settings to add the Alloy Origins as a device to Aurora.

I would use the newer Artemis instead of Aurora, with Razer Chroma / Wallpaper Engine, but Artemis's Chroma Grabber plugin only works for the keyboard and not the other OpenRGB devices (motherboard+fans, ram, gpu)

**Comment by xQwexx** [comment]
If openrgb is supporting your fn key then the problem probably is with aurora device layout, you can try my fork of Aurora where I am implemented a layout editor, so you could edit that key that it is controlled correctly, Here is a link for the build:
https://ci.appveyor.com/project/xQwexx/aurora/builds/41751390/artifacts

**Comment by tavallon** [comment]
> If openrgb is supporting your fn key then the problem probably is with aurora device layout, you can try my fork of Aurora where I am implemented a layout editor, so you could edit that key that it is controlled correctly, Here is a link for the build: https://ci.appveyor.com/project/xQwexx/aurora/builds/41751390/artifacts

Thanks for your fork! I tried it but I somehow broke my Synapse/OpenRGB/Aurora RGB integration for all devices except the keyboard. Same issue even on the newest dev build of Aurora and repairing Synapse. In the end I opened up Artemis and all my RGBs popped on. (whew) Decided to just go with Ambilight layer on Artemis, it's probably a better option for me anyways since it's less fuss and more customizable for my use. Thanks again
